core/src/main/java/bisq/core/app/BisqEnvironment.java

```diff
@@ -205,158 +205,74 @@ public BisqEnvironment(OptionSet options) {
                 options)));
     }
 
+    private String getProperty (PropertySource properties, String propertyKey, String defaultValue) {
```

Comment:
According to Uncle Bob it's good to keep public methods at the top, because they declare contract, while keeping private methods below.

Reply:
Sure, I moved it down. 

Although there is a real mess in this file - public and private methods are mixed together :)